minesweeper/use-cases/startGame/createBombs.ts

```diff
@@ -0,0 +1,30 @@
+import { BombModel } from '@minesweeper/domain/Bomb.model';
+
+const getRandomNumber = (max: number): number => Math.round(Math.random() * max);
+
+interface ICreateBombs {
+  rows: number;
+  columns: number;
+  bombsInput: number;
+}
+
+type TCreateBombs = (props: ICreateBombs) => BombModel[];
+
+export const createBombs: TCreateBombs = ({ bombsInput, rows, columns }) => {
+  const bombs: BombModel[] = [];
+
+  while (bombs.length < bombsInput) {
+    const newBomb: BombModel = {
+      position: [getRandomNumber(rows - 1), getRandomNumber(columns - 1)],
+    };
+
+    const haveSamePosition = bombs.some(
+      bomb => bomb.position[0] === newBomb.position[0] && bomb.position[1] === newBomb.position[1],
```

Comment:
I did a code similar to this, we can extract that in a common util

```
export   const samePosition = ({ position: [column, row] }: IPosition) =>
    column === columnMarked && row === rowMarked;
```

Reply:
I agree, create new issue/branch for this changes